themes/digital.gov/layouts/partials/core/blog-date-warning.html

```diff
@@ -1,11 +1,24 @@
 {{- $fiveYearsFromToday := now.AddDate -5 0 0 -}}
 {{- if lt .Date $fiveYearsFromToday -}}
-  <div class="usa-alert usa-alert--warning">
+  <div
+    class="usa-alert usa-alert--warning dg-alert dg-alert--out-of-date"
+    role="alert"
+  >
     <div class="usa-alert__body">
-      <p class="usa-alert__text">
-        This article was published more than 5 years ago. Links may not function
-        and information on the page may be out of date.
-      </p>
+      <h2 class="usa-alert__heading">
+        You have reached a collection of archived material
+      </h2>
+      <div class="usa-alert__text">
+        <p>
+          The content available is no longer being updated and as a result you
+          may encounter hyperlinks which no longer function.
+        </p>
+        <p>
+          You should also bear in mind that this content may contain text and
```

Comment:
```suggestion
          This content may contain text and
```

Reply:
It's best to avoid using "should" language, plus removing this language helps the alert be a bit more concise.

Reply:
Thanks for the suggestion, applied!